metron-platform/metron-integration-test/src/main/java/org/apache/metron/integration/EnrichmentIntegrationTest.java

```diff
@@ -164,6 +164,7 @@ public void test() throws Exception {
       setProperty("es.ip", "localhost");
       setProperty("index.date.format", dateFormat);
       setProperty("index.hdfs.output", hdfsDir);
+      setProperty("storm.topology.enrichment.workers", 1);
```

Comment:
I think this is what's failing the test. IIRC, that's setProperty(string, string).